### PR TITLE
Add a MOTD to the vagrant dev environment.

### PR DESCRIPTION
**Pull request by rbarlow**

### CODE REVIEW

File: playpen/vagrant-motd.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+  Similarly, run "workon pulp" to jump into the pulp platform virtualenv.
+* Each project has a "run-tests.py" to run the unit tests.
+
+More information can be found at
+https://pulp.readthedocs.org/en/latest/dev-guide/ and you can ask questions in
+#pulp on Freenode IRC.
+
+Happy hacking, and thanks for your contribution!
+
```
Comment (beav) [inline playpen/vagrant-motd.txt]: thanks for adding this additional whitespace before the prompt:smile: i forgot it in my original patch.

**Comment by beav** [comment]
LGTM for content assuming jenkins passes, but it looks like this may need a rebase before merging

**Comment by rbarlow** [comment]
ok test

**Comment by rbarlow** [comment]
ok test

**Comment by rbarlow** [comment]
Sweeeeeeet.

**Pull request merged**